### DISCUSSION TITLE
Configure to support only some components?

### DISCUSSION THREAD

**Discussion by khromov** [ssssota/svelte-exmarkdown#20] [Q&A] [answered]
Hi! Is it possible to configure `svelte-exmarkdown` to support only some components? For example, I want to support only bold/italic syntax and nothing else. Everything else should be rendered as-is (as text).

**Comment by ssssota** [comment, marked as answer]
Thank you for asking!

You can like this: https://stackblitz.com/edit/svelte-exmarkdown-only-bold-italic

I have not verified this, but I think the remark/rehype-plugin can be used to do the same thing too.

**Reply by khromov** [reply]
Thank you!
